stormpath/express-stormpath
# Fix confusing setup docs

Fix confusing setup docs.
#### How to verify

Check the changes and verify that it solves the problem described in the issue.

Fix #490
